app/views/templates/base.volt

```diff
@@ -37,6 +37,14 @@
                 </div>
                 <div class="m-t-md m-b-lg" id="articles">
                     <div class="article-content">
+                        <div>
+                            {% if language == 'en' %}
+                                <a class="article-content-edit-link pull-right" href="https://github.com/phalcon/docs/blob/{{ version }}/{{ language }}/{{ page == null ? '': page ~".md"}}">Edit this page on GitHub</a>
```

Comment:
`if language is 'en'`:

`/{{ version }}/{{ language }}/` => `/{{ version }}/en/`